src/blenderbim/blenderbim/bim/module/search/prop.py

```diff
@@ -16,10 +16,13 @@
 # You should have received a copy of the GNU General Public License
 # along with BlenderBIM Add-on.  If not, see <http://www.gnu.org/licenses/>.
 
-import bpy
```

Comment:
I would not remove this line, sometimes bpy is highlighted as not used directly by the IDE but it may be used in an expression or in a test in the script

Reply:
Ok good to know!